Makefile

```diff
@@ -1,7 +1,8 @@
+DOCKER_REGISTRY   ?=
+IMAGE_PREFIX      ?= ciscosso
```

Comment:
In some of my repos, I use the .envrc / direnv pattern to split out the project specific stuff from an otherwise boilerplate makefile.  Not sure if it would would be better here. So `May` on that.

Reply:
I'm gonna let this be because I'm rebasing someone else's code :)